alot/settings/manager.py

```diff
@@ -91,8 +91,8 @@ def read_config(self):
         hooks_path = os.path.expanduser(self._config.get('hooksfile'))
         try:
             self.hooks = imp.load_source('hooks', hooks_path)
-        except:
-            logging.debug('unable to load hooks file:%s', hooks_path)
+        except Exception as e:
```

Comment:
You don't need the variable name when you use logging.exception() and hence you also don't need the class specification.

Reply:
I removed it